src/main/java/org/sagebionetworks/bridge/models/schedules2/adherence/SessionCompletionState.java

```diff
@@ -22,8 +22,8 @@
     DECLINED;
 
     // These include all sessions except sessions that are not yet available, and thus cannot be included
-    // in a calculation of adherence. 
+    // in a calculation of adherence.
     public static final EnumSet<SessionCompletionState> NONCOMPLIANT = EnumSet.of(ABANDONED, EXPIRED, DECLINED);
     public static final EnumSet<SessionCompletionState> COMPLIANT = EnumSet.of(COMPLETED);
-    public static final EnumSet<SessionCompletionState> UNKNOWN = EnumSet.of(UNSTARTED, STARTED);
+    public static final EnumSet<SessionCompletionState> UNKNOWN = EnumSet.of(UNSTARTED, STARTED, NOT_YET_AVAILABLE);
```

Comment:
So this makes sense for *weekly* adherence, but what about *study-scoped* adherence?